src/main/java/frc/robot/commands/RunShooter.java

```diff
@@ -0,0 +1,82 @@
+// RobotBuilder Version: 3.1
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// Java from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+import frc.robot.subsystems.Shooter;
+
+// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+
+/**
+ *
+ */
+public class RunShooter extends CommandBase {
+
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
+    private final Shooter m_shooter;
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
+
+    private Joystick buttonBox;
+
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+
+    public RunShooter(Shooter subsystem) {
+
```

Comment:
@drakeot probably should've clarified in the task, need all the challenges in one code base. It appears you took the command used for challenge 2 and repurposed it for challenge 3, losing the previous buttons and command signature:

```diff
21750709dc7289fff22abef2de499f3662c4f2b2
-public RunShooter(double speed, Shooter subsystem) {
+public RunShooter(Shooter subsystem) {

Reply:
Ah, yes, I did. I can go back and split them into different commands.

Reply:
Your choice. The nice part is the history is there, so can see you managed to do Challenge 2 successfully at some point.